CHANGELOG.md

```diff
@@ -313,7 +321,9 @@ and this project adheres to [Semantic Versioning][semver].
 [keepachangelog]: https://keepachangelog.com/en/1.0.0/
 [semver]: https://semver.org/spec/v2.0.0.html
 
-[Unreleased]: https://github.com/openlawlibrary/pygls/compare/v0.11.3...HEAD
+[Unreleased]: https://github.com/openlawlibrary/pygls/compare/v0.12-rc1...HEAD
```

Comment:
Duplicate line

Reply:
I think that's just the way the diff looks?

Reply:
no, you have two 'Unreleased' entries

Reply:
Ah yes! Sorry, I hadn't pulled those changes locally. I've pushed now again changing `0.12-rc2` to just `0.12`.

Reply:
you seem actually to have changed to `0.12-rc1`, but close enough I guess

Reply:
uff damn that's a shame. thanks for pointing that out